File: Frontend/Logic/B20.Tests/Architecture/Events/Context/Fixtures.cs
using B20.Architecture.Contexts.Api;
using B20.Architecture.Events.Fixtures;
using B20.Events.Api;

namespace B20.Tests.Architecture.Events.Context
{
    public class EventsMocks: ContextModule
    {
        public void Apply(ContextBuilder builder)
        {
            builder
                .SetImpl<EventPublisher, EventPublisherMock>();
        }
    }
}

File: Backend/lib/src/testFixtures/kotlin/com/github/bratek20/ostium/carddrawing/context/Fixtures.kt
package com.github.bratek20.ostium.carddrawing.context

import com.github.bratek20.architecture.context.api.ContextBuilder
import com.github.bratek20.architecture.context.api.ContextModule
import com.github.bratek20.ostium.carddrawing.api.CardDrawerFactory
import com.github.bratek20.ostium.carddrawing.fixtures.CardDrawerFactoryMock

class CardDrawerMocks: ContextModule {
    override fun apply(builder: ContextBuilder) {
        builder.setImpl(CardDrawerFactory::class.java, CardDrawerFactoryMock::class.java)
    }
}